.gitlab/deploy_containers/deploy_containers_a7.yml

```
@@ -354,3 +354,13 @@ deploy_containers_latest-a7-fips:
       - JMX:
           - ""
           - "-jmx"
+
+deploy_containers-single_machine_performance-full-amd64-a7:
+  extends: .docker_publish_job_definition
+  stage: deploy_containers
+  rules:
+    !reference [.on_internal_final]
```

I expect this should match the rules for the regression detector job which is expected to use this image.

```suggestion
    - !reference [.on_dev_branches]
    - when: on_success
```

---

This seems to create an issue for [compute_gitlab_ci_config](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/891684145)